### PR TITLE
Feat: make poller work for an arbitrary number of schemas (users)

### PR DESCRIPTION
**Pull request by hellypi**
Commits related to this PR start (and include) from a4a77e8
For code review, relevant files changed are app.js and poller.js

### CODE REVIEW

**Comment by hellypi** [comment]
[Change the lamassu-server poller to get it ready for pazuz (lamassu-side)](https://trello.com/c/QuZrNDvI/724-change-the-lamassu-server-poller-to-get-it-ready-for-pazuz-lamassu-side)

File: lib/poller.js [hunk 1]
```diff
@@ -150,4 +150,8 @@ function setup (__settings = null, schemasToAdd = [], schemasToRemove = []) {
   return initializeEachSchema(schemasToAdd)
 }
 
-module.exports = { setup, reload }
+function getActiveSchemas () {
+  return Array.from(schemaCallbacks.keys())
```
Comment (chaotixkilla) [inline lib/poller.js]: Couldn't this be just `return _.keys(schemaCallbacks)`?

**Review by chaotixkilla — CHANGES_REQUESTED** [review]
Has conflicts

**Comment by hellypi** [comment]
PR replaced by https://github.com/lamassu/lamassu-server/pull/709

File: lib/poller.js (hunk 1 above)
Comment (hellypi) [reply lib/poller.js]: Since it's a Map lodash doesn't work on it. And I want to return an array not a map iterator so I also have to use Array.from

**Pull request closed without merging**